packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -33,35 +34,56 @@ const generateAssetData = async () => {
     ...osmosisAssets,
     ...avalancheAssets,
     ...optimismAssets,
+    ...bnbsmartchainAssets,
   ]
+
   // remove blacklisted assets
   const filteredAssetData = filterOutBlacklistedAssets(unfilteredAssetData)
 
-  // For coins not currently in the color map, check to see if we can generate a color from the icon
-  const filteredWithColors = await setColors(filteredAssetData)
-
   // deterministic order so diffs are readable
-  const orderedAssetList = orderBy(filteredWithColors, 'assetId')
+  const orderedAssetList = orderBy(filteredAssetData, 'assetId')
 
   const ethAssetNames = ethAssets.map((asset) => asset.name)
   const avalancheAssetNames = avalancheAssets.map((asset) => asset.name)
   const optimismAssetNames = optimismAssets.map((asset) => asset.name)
+  const bnbsmartchainAssetNames = bnbsmartchainAssets.map((asset) => asset.name)
 
   const generatedAssetData = orderedAssetList.reduce<AssetsById>((acc, asset) => {
     const { chainReference } = fromAssetId(asset.assetId)
 
+    // mark any ethereum assets that also exist on other evm chains
+    if (
+      chainReference === CHAIN_REFERENCE.EthereumMainnet &&
+      asset.assetId !== ethAssetId &&
+      avalancheAssetNames
+        .concat(optimismAssetNames)
+        .concat(bnbsmartchainAssetNames)
```

Comment:
If you're feeling spready, this is turbo nitpicky but would be easier to visually grasp that these 3 lines are one expression, and the next is checking for inclusion over another expression but realistically doesn't matter

Reply:
https://github.com/shapeshift/lib/pull/1197/commits/2723dc3d169c7e25428aca3a4fe7a1865d774989